src/actions/actions.ts

```diff
@@ -2694,3 +2694,76 @@ class ToggleCaseAndMoveForward extends BaseMovement {
     return position.getRight();
   }
 }
+
+abstract class IncrementDecrementNumberAction extends BaseMovement {
+  modes = [ModeName.Normal];
+  canBePrefixedWithCount = true;
+
+  offset: number;
+
+  public async execActionWithCount(position: Position, vimState: VimState, count: number): Promise<Position> {
+    count = count || 1;
+    const text = TextEditor.getLineAt(position).text;
+
+    // They may start on a number, so check the current word boundaries first.
+    let wordEnd = position.getCurrentWordEnd(true);
```

Comment:
If a user is insane enough to define numbers as word separators, these motions make no semantic sense (is 1234 4 separate words or one? If we select the whole thing with visual mode and do ctrl a, what should happen?) 

I'm completely fine with this being undefined behavior.